src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -124,8 +156,17 @@ export default compose(
     withLocalize,
     withReportAndReportActionOrNotFound,
     withOnyx({
+        allTransactions: {
+            key: ONYXKEYS.COLLECTION.TRANSACTION,
+        },
+        draftSplitTransactions: {
+            key: ONYXKEYS.COLLECTION.DRAFT_SPLIT_TRANSACTION,
+        },
```

Comment:
We are subscribing to the whole collections. We only need to subscribe to one key here. e.g: 

```js
        transaction: {
            key: ({route, reportActions}) => {
                const reportAction = reportActions[`${route.params.reportActionID.toString()}`];
                return `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(reportAction, 'originalMessage.IOUTransactionID', 0)}`;
            },
        },

```

Reply:
Done